Remove unneeded call to rounddown_pow_of_two function when initializing variable.

--- a/mm/memory.c
+++ b/mm/memory.c
@@ -2770,8 +2770,7 @@ void do_set_pte(struct vm_area_struct *v
 	update_mmu_cache(vma, address, pte);
 }
 
-static unsigned long fault_around_bytes __read_mostly =
-	rounddown_pow_of_two(65536);
+static unsigned long fault_around_bytes __read_mostly = 65536;
 
 #ifdef CONFIG_DEBUG_FS
 static int fault_around_bytes_get(void *data, u64 *val)
